lib/src/modules/pipeline.dart

```diff
@@ -101,7 +101,7 @@ class Pipeline {
       List<Logic> signals = const [],
       Map<Logic, Const> resetValues = const {},
       this.reset}) {
-    _stages = stages.map((stage) => _PipeStage(stage)).toList();
+    _stages = stages.map(_PipeStage.new).toList();
```

Comment:
[About constructor tear-offs](https://medium.com/dartlang/dart-2-15-7e7a598e508a).

Reply:
This is just an explanation. I wanted to add a comment to the commit, but it turned out like a review.